modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceSelector.js

```diff
@@ -120,6 +121,11 @@ const ExperienceSelector = ({
 	const modalExperienceStateRef = useRef(modalExperienceState);
 	modalExperienceStateRef.current = modalExperienceState;
 
+	const isSelectedExperienceActive = experiences.find(
+		(exp) =>
```

Comment:
can we call this local var `experience`?
or maybe just destructuring `{segmentsExperienceId}` so we don't have to name it?

Reply:
Let me wait for the ci and if it fails I destructure the variable.